$MirOS: ports/net/mcabber/patches/patch-aclocal_m4,v 1.1 2008/06/25 18:16:58 tg Exp $

	evil kludge, but hey

--- aclocal.m4.orig	Sun Oct  5 20:34:59 2008
+++ aclocal.m4	Sun Oct  5 20:35:07 2008
@@ -56,7 +56,7 @@ AC_ARG_ENABLE(glibtest, [  --disable-gli
       esac
   done
 
-  PKG_PROG_PKG_CONFIG([0.16])
+  PKG_PROG_PKG_CONFIG([0.15])
 
   no_glib=""
 
